app/poros/food_list.rb

```diff
@@ -0,0 +1,9 @@
+class FoodList
+  attr_reader :total_hits,
+              :foods
+
+  def initialize(food_hash)
+    @total_hits = food_hash[:totalHits]
+    @foods = food_hash[:foods].first(10).map { |food| FoodItem.new(food) }
+  end
+end
```

Comment:
I really like this idea, George. Because ideally, we're just sending one object to the view, and you're combining everything we need for the view right here in this one object. You could also consider using a parameter in your API call to limit how many foods you get back for that first page, but I'm also okay with this, as you're limiting it to the first ten before iterating and creating the FoodItem. This is a chunk of logic in your initialize method though, so I'd also suggest putting this line 7 into a method instead.